src/gatsby/src/components/advisories/advisoryFilter.js

```diff
@@ -55,6 +55,14 @@ const AdvisoryFilter = ({
       navigate("/active-advisories")
     }
   }
+  const filterBy = (option, props) => {
+    const input = props.text.toLowerCase()
+    if (input === "all") {
+      return true
+    } else {
+    return option.label.toLowerCase().includes(input)
+    }
```

Comment:
Line 63 is missing an indent here. I noticed it doesn't indent when I save files in this repo; is there no auto-formatting at all?

Reply:
No I'm just using the Prettier on my end 😅 I think we should have done setting globally... maybe we can talk about it when Mike's back.

Reply:
I saw the gatsby folder has `npm run format` to do it manually, but setting it up to format automatically would probably save you a lot of effort 👍  We can chat about it any time you want to look into it